src/main.qml

```diff
@@ -127,6 +128,7 @@ Application {
                                 app.height*0.29 :
                                 app.height*0.33 :
                             app.height*0.06
+        font.styleName: app.bpm / 100 >= 1 ? "SemiCondensed" : ""
```

Comment:
Can't you just use the check: `app.bpm >= 100 ?  "SemiCondensed" : ""`?

Reply:
Wow, my brain went extra steps again.